website/docs/r/app_service_virtual_network_connection.html.markdown

```diff
@@ -0,0 +1,169 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_app_service_virtual_network_connection"
+sidebar_current: "docs-azurerm-resource-app-service-virtual-network-connection"
+description: |-
+  Manages Gateway Required Virtual Network Connection (Integrate with an App Service).
+
+---
+
+# azurerm_app_service_virtual_network_connection
+
+Manages Gateway Required Virtual Network Connection (Integrate with an App Service).
+This function is Based on script https://gallery.technet.microsoft.com/scriptcenter/Connect-an-app-in-Azure-ab7527e3
+
+To configure a gateway required virtual network integration, please make sure the virtual network already have a gateway configured with point-to-site addresses.
+Please refer to this doc: https://docs.microsoft.com/en-us/azure/app-service/web-sites-integrate-with-vnet#gateway-required-vnet-integration

```

Comment:
Can this not be set via terraform?

Reply:
Would we show the terraform required to do this in the example and maybe make this a note?